docs/preview features/orchestrator.md

```diff
@@ -0,0 +1,136 @@
+# Orchestrator Preview
+Orchestrator is a powerful new language model that is optimized for conversational AI applications.
+
+- [Technical Overview](https://github.com/microsoft/botframework-sdk/blob/main/Orchestrator/docs/Overview.md)
+- [Samples](https://github.com/microsoft/BotBuilder-Samples/tree/main/experimental/orchestrator)
+
+## Orchestrator Features
+- Easy to use - Use your existing LU files from LUIS without modification
+- State of the Art - Extensive pre-trained transformer-based model for conversational applications
+- Extremely fast Inference Time: Written in C++ for performance
+- Cross-platform Support: Runs on Windows, Linux and Mac OS X, including Cloud platforms
+- Free to use: Language models are deployed alongside your bot, there are no usage fees
+
+## Getting Started
+------------------
+### Limitations
+For this preview release, please be aware of these limitations:
+
+1. Orchestrator has to be deployed on an x64 platform. If you are deploying your bot to a cloud service, please be aware that you need to publish your bot with `win-x64` enabled, and enable x64.
+
+1. We are supporting English-only for this release.
+
+1. Orchestrator does not support Entity extraction for this release. Any Entities declared in your LU file will be ignored.
+
+Please see the [roadmap](https://github.com/microsoft/botframework-sdk/blob/main/Orchestrator/docs/Overview.md#Roadmap) for when these features will be added to Orchestrator.
+
+
+### Adding Orchestrator to a new bot in Composer
+1. Enable the preview feature in Orchestrator:
+    - Navigate to `Application Settings` (
+    <img src="orchestrator-assets/application-settings.png" alt="drawing" width="30"/> icon)
+    - Enable Orchestrator preview feature under `Applicatio Updates -> Preview features`
+
+      ![orchestrator flight](orchestrator-assets/preview-flag-orchestrator.png)
+
+1. Create a new bot, or open an existing bot project. Select a dialog in the bot project that you would like to try Orchestrator with. Select `Orchestrator recognizer` from the `Recognizer Type` dropdown
```

Comment:
"Create a new bot or open an existing bot project."
"Select `Orchestrator recognizer` from the `Recognizer Type` drop-down menu."